documentation/docs/specs/revocation_bitmap_2022.md

```diff
@@ -0,0 +1,182 @@
+---
+title: RevocationBitmap2022 Specification
+sidebar_label: Revocation Bitmap Specification
+description: The specification for the embedded revocation bitmap.
+image: /img/Identity_icon.png
+keywords:
+  - DID
+  - specs
+  - specifications
+  - revocation
+  - bitmap
+---
+
+# RevocationBitmap2022
+
+## Abstract
+
+This specification describes an on-tangle mechanism for publishing the revocation status of Verifiable Credentials embedded in an issuer's DID document.
```

Comment:
Maybe we should link the first occurrence.
```suggestion
This specification describes an on-tangle mechanism for publishing the revocation status of [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/#dfn-verifiable-credentials) embedded in an issuer's DID document.
```

Reply:
I'll link our [own VC](https://wiki.iota.org/identity.rs/concepts/verifiable_credentials/overview) section of the wiki, if that's okay.